daprdocs/content/en/developing-applications/building-blocks/distributed-lock/howto-use-distributed-locks.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Remove this entire file, it's a duplicate of the one above.

Reply:
removed